Source/CombatExtended/CombatExtended/Projectiles/ProjectileCE.cs

```diff
@@ -390,8 +390,11 @@ public override void ExposeData()
         }
         #endregion
 
-        public virtual void RayCast(Thing launcher, VerbProperties verbProps, Vector2 origin, float shotAngle, float shotRotation, float shotHeight = 0f, float shotSpeed = -1f, float spreadDegrees = 0f, Thing equipment = null) {
-            const float magicLaserDamageConstant = 1789.255934470907f;
+        public virtual void RayCast(Thing launcher, VerbProperties verbProps, Vector2 origin, float shotAngle, float shotRotation, float shotHeight = 0f, float shotSpeed = -1f, float spreadDegrees = 0f, float aperatureSize = 0.03f, Thing equipment = null) {
+
+            float magicSpreadFactor = Mathf.Sin(0.06f / 2 * Mathf.Deg2Rad) + aperatureSize;
```

Comment:
Could you add a comment explaining where this comes from?
https://i.pinimg.com/originals/61/9f/5e/619f5e462f2099c470486e589e748816.gif